src/Core/Settings.cpp

```diff
@@ -2067,6 +2067,9 @@ Write exception in output format to produce valid output. Works with JSON and XM
 )", 0) \
     DECLARE(UInt64, http_response_buffer_size, 0, R"(
 The number of bytes to buffer in the server memory before sending a HTTP response to the client or flushing to disk (when http_wait_end_of_query is enabled).
+)", 0) \
+    DECLARE(Bool, http_exception_tagging, false, R"(
```

Comment:
Let's also test if we can enable it for a readonly user

Reply:
I am also curious: does it throw if it is set on an older CH version, e.g. do we need to track CH version on the language client side before setting it?

Reply:
Had a chat with @slabko and @tavplubix. We decided not to have this settings at all.

Given, we made sure we have a `\r\n` before between exception marker and a tag. It wouldn't break any existing behavior and this feature can be enabled by default without needing any toggle.

I will remove the this setting.